pyproject.toml

```diff
@@ -18,6 +18,11 @@ packages = [{ include="cognite", from="." }]
 # let scripts use print statements
 "scripts/*" = ["T201"]
 
+[tool.ruff.format]
+# format code examples in docstrings to avoid users scrolling horizontally
+docstring-code-format = true
+docstring-code-line-length = 85
```

Comment:
This is the PR, the rest is auto-fmt-work

Reply:
Doesn't ruff already parse the `pyproject.toml` file already? I tried to check out the first commit in the PR and run `pre-commit --files SOME_PYTHON_FILE.PY`, and it did format the docstrings. If I do the same on the master branch, the docstrings are untouched.

Reply:
If I checkout `f1c44291` and then run `pre-commit run --all`, I end up with the same state as `b57a65f1` :heavy_check_mark: